tests/integration/test_issue_2001.py

```diff
@@ -0,0 +1,37 @@
+import os.path
+import subprocess
+import sys
+
+from pex.testing import PY38, ensure_python_interpreter, make_env, run_pex_command
+from pex.typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any
+
+
+def test_compile_error_as_warning(
+    tmpdir,  # type: Any
+):
+    # type: (...) -> None
+    pex = os.path.join(str(tmpdir), "pex.pex")
+    run_pex_command(
+        args=[
+            "--include-tools",
+            "aenum",
+            "-o",
+            pex,
+        ],
+        # The package script requires Python 3.
```

Comment:
Is that true? I don't see any 3-isms here: https://github.com/ethanfurman/aenum/blob/master/setup.py (they don't use tags or branches for releases so I'm just guessing really!).

Reply:
This is just me doing copy paste.